pkg/util/timeutil/lowercase/main.go

```diff
@@ -49,11 +48,21 @@ var lowercaseTimezones = map[string]string{
 `
 
 func main() {
+	var filenameFlag = flag.String("filename", "lowercase_timezones.go", "filename path")
+	var zoneInfo = flag.String("zoneinfo", "zoneinfo.zip", "zoneinfo.zip path")
+	var zipdataFile *os.File
```

Comment:
This is a pretty wild default value for the flag. :) 

I would just make the default value `""`, then you can check if the string is empty and set it to `GOROOT/lib/time/zoneinfo.zip` if so.

Reply:
Done =)